TELCODOCS-1506 Telco CORE Reference Design Specification (#65732)
openshift/openshift-docs: modules/telco-core-power-management.adoc

// Module included in the following assemblies:
//
// * telco_ref_design_specs/ran/telco-core-ref-components.adoc

:_content-type: REFERENCE
[id="telco-core-power-management_{context}"]
= Power Management

New in this release::

> * Ability to specify a maximum latency that is C-state for a low latency pod when using per-pod power management. Previously, C-states could only be disabled completely on a per pod basis.

Reviewer: ```suggestion
* You can specify a maximum latency that is C-state for a low latency pod when using per-pod power management. Previously, C-states could only be disabled completely on a per pod basis.
```

// https://issues.redhat.com/browse/CNF-7760

Description::

> The https://docs.openshift.com/container-platform/4.14/rest_api/node_apis/performanceprofile-performance-openshift-io-v2.html#spec-workloadhints[Performance Profile] can be used to configure a cluster in a high power, low power or mixed (https://docs.openshift.com/container-platform/4.14/scalability_and_performance/cnf-low-latency-tuning.html#node-tuning-operator-pod-power-saving-config_cnf-master[per-pod power management]) mode. The choice of power mode will depend on the characteristics of the workloads running on the cluster particularly how sensitive they are to latency.

Reviewer: ```suggestion
The https://docs.openshift.com/container-platform/4.14/rest_api/node_apis/performanceprofile-performance-openshift-io-v2.html#spec-workloadhints[Performance Profile] can be used to configure a cluster in a high power, low power or mixed (https://docs.openshift.com/container-platform/4.14/scalability_and_performance/cnf-low-latency-tuning.html#node-tuning-operator-pod-power-saving-config_cnf-master[per-pod power management]) mode. The choice of power mode depends on the characteristics of the workloads running on the cluster particularly how sensitive they are to latency.
```

Limits and requirements::

> * Power configuration relies on appropriate BIOS configuration for example enabling C-states and P-states which varies between hardware vendors.

Reviewer: ```suggestion
* Power configuration relies on appropriate BIOS configuration, for example, enabling C-states and P-states. Configuration varies between hardware vendors.
```

> 

Reviewer: remove empty line
